### DISCUSSION TITLE
Unable to access HSGQ SFP web interface on RB5009

### DISCUSSION THREAD

**Discussion by Strykar** [Anime4000/RTL960x#239] [Q&A] [answered]
I can ping the IP I set on the SFP, `192.168.1.2`, but not `192.168.1.1`.
I can only ping it from the RB5009, not from the LAN.
```
[admin@MikroTik] > ip/address/print 
Columns: ADDRESS, NETWORK, INTERFACE
# ADDRESS          NETWORK       INTERFACE   
;;; defconf
0 192.168.88.1/24  192.168.88.0  bridge      
1 192.168.1.2/24   192.168.1.0   sfp-sfpplus1
```

**Comment by Anime4000** [comment]
You need do NAT

for me, 
`pppoe-out1` in WAN interface group
`sfp-sfpplus1` in WAN interface group

masquerade WAN interface group

simple as that

**Reply by Strykar** [reply]
![image](https://github.com/Anime4000/RTL960x/assets/2946372/c3668f69-1325-42b5-b373-6d477cc7f885)

I've tried with and without the `src / dst` ranges in the NAT rule but I still can't ping `192.168.1.1`.
Does this need the fiber plugged in first?

**Reply by vuducdong** [reply]
Which version of RouterOS are you using? In the settings on the sfp port, what mode are you in? 1G, 2.5G ??? And on HSQG, is LAN_SDS set to 1 or has it been changed to it?

I see the sfp status is RX Loss, so of course you can't ping sfp.

**Reply by Strykar** [reply]
ROS ver 7.8 and the SFP port was set to auto-negotiate.

I have tried plugging in the fiber, unfortunately it does not detect a link (`Rx loss` remains checked).
Swapping the fiber back into the ISP router, the Fiber link light on the ISP router and Internet are restored instantly.

Things I've tried:
- Set the sfp's IP: `192.168.1.2 / 255.255.255.0`
- Changed the `sfp` MAC address on the RB5009 to the one my ISP router's optical link uses, that did not change anything.
- Tried manually selecting the speeds from 10M to 2.5G, it did not help.

Unforunately, I can't even ping the HSGQ, much less run commands on it to see what `LAN_SDS` is set to.
Kinda lost here :(

**Reply by Strykar** [reply]
@Anime4000 After your suggestion, I can ping 192.168.1.2 from the LAN too, but neither the RB5009 nor anything from the LAN can ping 192.168.1.1.

If this is indeed a case of the optical link required to bring up the 192.168.1.1 interface on the SFP, what are my options since it won't detect the link?

**Reply by Anime4000** [reply]
I think since ROS 7.8 need fiber to be plugged then can accsss.

maybe try downgrade to 7.7

**Reply by Strykar** [reply]
I think the fact that no link is ever detected with both SFP modules, indicates something else. I have ordered an SC/APC to SC/UPC cable to test with UPC instead. It should be here in a day or two.

**Comment by Strykar** [comment]
Just as I thought, the SFP module is UPC.
Now I do have the fiber link up, and in spite of setting `192.168.1.2` to the SFP interface, I am unable to ping `192.168.1.1`, I also do not see a route added for it. I can however ping `192.168.1.2` from the router itself.

![image](https://github.com/Anime4000/RTL960x/assets/2946372/fa8da78b-d788-4579-8a00-06815ed356c8)

**Reply by vuducdong** [reply, marked as answer]
@Strykar 

1, you can set the same:
![image](https://github.com/Anime4000/RTL960x/assets/74129390/72de2323-5867-4987-acd4-d810d3ebe31d)

2, and src-nat

![image](https://github.com/Anime4000/RTL960x/assets/74129390/932060e2-1e91-4f08-9e81-a19d076f3cc1)

![image](https://github.com/Anime4000/RTL960x/assets/74129390/44a70f23-f6f7-47ab-bcb8-be5b53ed7eb3)

3, and finally, when you are in LAN bridge 192.168.88.1/24, you can ping to 192.168.1.1 and access sfp via web and telnet

If it still doesn't work, you can add:
![image](https://github.com/Anime4000/RTL960x/assets/74129390/7dfe7eda-0bb7-4e7c-a0df-f4ca83169c31)

**Reply by Strykar** [reply]
Thank you very much, I can now ping and access the SFP web interface! Did not need step 3.
I will figure out the rest on the Indian ISP forum, cheers.
![Screenshot from 2024-01-23 17-44-54](https://github.com/Anime4000/RTL960x/assets/2946372/55cebd02-1633-45d4-9ec9-80269a7c4566)
![Screenshot from 2024-01-23 17-44-48](https://github.com/Anime4000/RTL960x/assets/2946372/183ddeb4-2de4-444f-b700-f35a7cbe1da2)

**Reply by Strykar** [reply]
@vuducdong I have changed the default HSGQ SFP's web password.
Could this web interface be accessed from any WAN interface of the SFP?

**Reply by vuducdong** [reply]
I don't get what you mean. If you want to access SFP's website remotely (from outside), you just need port forwarding. On Mikrotik it's called dst-nat.
